src/js/features/content/content.store.ts

```diff
@@ -1,25 +1,32 @@
 import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
 import axios from 'axios';
-import { aboutUrl } from '@/constants/contentConstants';
+import { partialAboutUrl } from '@/constants/contentConstants';
 import { printAPIError } from '@/utils/error.util';
 
-export const makeGetAboutRequest = createAsyncThunk<string, void, { rejectValue: string }>(
-  'content/getAboutHTML',
-  (_, { rejectWithValue }) =>
-    axios
-      .get(aboutUrl)
-      .then((res) => res.data)
-      .catch(printAPIError(rejectWithValue))
-);
+export const makeGetAboutRequest = createAsyncThunk('content/getAboutHTML', async () => {
+  const en_aboutHTML = await axios
+    .get(`${partialAboutUrl}en_about.html`)
+    .then((res) => res.data)
+    .catch(printAPIError);
+
+  const fr_aboutHTML = await axios
+    .get(`${partialAboutUrl}fr_about.html`)
+    .then((res) => res.data)
+    .catch(printAPIError);
+
+  return { en_aboutHTML, fr_aboutHTML };
+});
 
 export type ContentState = {
   isFetchingAbout: boolean;
-  aboutHTML: string;
+  en_aboutHTML: string;
+  fr_aboutHTML: string;
 };
```

Comment:
I would change the ContentState shape for:
```typescript
export type ContentState = {
  isFetchingAbout: boolean;
  about: {[key: string]: string}
};

const initialState: ContentState = {
  isFetchingAbout: true,
  about: {
    en: "",
    fr: ""
  }
};
```
This way, we can get the about content by key and avoid the if blocks in PublicOverview's useEffect:
```typescript
  const { isFetchingAbout, about } = useAppSelector((state) => state.content);

  const [aboutContent, setAboutContent] = useState('');

  useEffect(() => {
    const activeLanguage = i18n.language;
    const activeAbout = about[activeLanguage];
    setAboutContent(activeAbout);
  }, [i18n.language, about]);
```
Like this we won't have to change the PublicOverview component if we add new languages